src/utils/wallet.ts

```diff
@@ -10,7 +14,11 @@ export const ADDRESS_LENGTH = 40
  * @param password password to encrypt the wallet
  */
 export async function mnemonicToV3(mnemonic: string, password: string): Promise<V3Keystore> {
-  return JSON.parse(await Wallet.fromMnemonic(mnemonic).encrypt(password))
+  const parsed = JSON.parse(await Wallet.fromMnemonic(mnemonic).encrypt(password))
+  const seed = Utils.hexToBytes(utils.mnemonicToSeed(mnemonic).replace('0x', '')) as Seed
+  parsed[X_SEED_KEY] = encryptSeed(seed, password)
```

Comment:
why is it necessary? the seed should be retreivable from the v3 wallet after decrypting it and initialating it with `v3ToWallet`